addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/TechPassiveScanner.java

```diff
@@ -433,7 +433,9 @@ Builder createAlert(String url, ApplicationMatch appMatch) {
                 .setConfidence(Alert.CONFIDENCE_MEDIUM)
                 .setUri(url)
                 .setDescription(getDesc(app))
-                .setCweId(200)
+                .setCweId(
+                        497) // CWE-497: Exposure of Sensitive System Information to an Unauthorized
```

Comment:
Not sure we are checking for sensitive system nformation but if there's no better ID.

Reply:
Maybe that's better.